packages/host/app/components/operator-mode/card-schema-editor.gts

```diff
@@ -103,28 +110,66 @@ export default class CardSchemaEditor extends Component<Signature> {
         height: 20px;
         width: 20px;
       }
+
+      .header {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+      }
+
+      .total-fields {
+        display: flex;
+        align-items: baseline;
+        gap: var(--boxel-sp-xxxs);
+        margin-left: auto;
+      }
+
+      .total-fields > * {
+        margin: 0;
+      }
+
+      .total-fields-value {
+        font: 600 var(--boxel-font);
+      }
+
+      .total-fields-label {
+        font: var(--boxel-font-sm);
+      }
     </style>
 
     <div
       class='schema-editor-container'
       data-test-card-schema={{@cardType.displayName}}
     >
-      <div class='pill'>
-        <div class='realm-icon'>
-          <RealmInfoProvider @fileURL={{@cardType.module}}>
-            <:ready as |realmInfo|>
-              <img
-                src={{realmInfo.iconURL}}
-                alt='Realm icon'
-                data-test-realm-icon-url={{realmInfo.iconURL}}
-              />
-            </:ready>
-          </RealmInfoProvider>
+      <div class='header'>
+        <div class='pill'>
+          <div class='realm-icon'>
+            <RealmInfoProvider @fileURL={{@cardType.module}}>
+              <:ready as |realmInfo|>
+                <img
+                  src={{realmInfo.iconURL}}
+                  alt='Realm icon'
+                  data-test-realm-icon-url={{realmInfo.iconURL}}
+                />
+              </:ready>
+            </RealmInfoProvider>
+          </div>
+          <div>
+            <span>
+              {{@cardType.displayName}}
+            </span>
+          </div>
         </div>
-        <div>
-          <span>
-            {{@cardType.displayName}}
-          </span>
+        <div class='total-fields' data-test-total-fields>
+          {{#if (gt this.totalOwnFields 0)}}
+            <p class='total-fields-value'>+ {{this.totalOwnFields}}</p>
```

Comment:
I think `<p>`s should be used only for paragraphs (series of sentences).

Reply:
Updated [b5ebe79](https://github.com/cardstack/boxel/pull/701/commits/b5ebe79590e5ecb1f0ef0360f78be7ee77748783)